assets/stylesheets/generics/colors.scss

```diff
@@ -9,7 +9,7 @@
   --alto: #dadada;
   --gray: #929292;
   --mariner: #2f81cd;
-
+  --mine-shaft: #333333;
```

Comment:
we need not add these colors for now, let it be the default colors

Reply:
Ok.. Actually there is no good color to suit this purpose. Suggest we keep this one

Reply:
that, the designers will take a call, they have their own set of colors